### DISCUSSION TITLE
How to enable include mode without using UI

### DISCUSSION THREAD

**Discussion by cyberumut** [zwave-js/zwave-js-ui#3529] [Q&A] [answered]
how to enable include mode without using web server. 
i dont want use web panel is there any method include mode and exclude mode without web server ?

**Comment by robertsLando** [comment, marked as answer]
MQTT: https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=startinclusion

Websocket: https://github.com/zwave-js/zwave-js-server (Settings > Home Assistant > Ws Server)

**Comment by cyberumut** [comment]
> MQTT? Websocket?

how to enable include mode with MQTT  ?

**Comment by robertsLando** [comment]
I have updated my comment with links

**Comment by cyberumut** [comment]
> I have updated my comment with links
thank u sir

**Comment by robertsLando** [comment]
The zwave-js server can be enabled from settings under home assistant section (you can also customize port).

The controller commands using websockets can be found here https://github.com/zwave-js/zwave-js-server#controller-level-commands
